<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgNote</class>
 <widget class="QWidget" name="CDlgNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBoxList">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox#groupBoxList
{
	border:1px solid gray;
	border-radius:10px;
	background-color: rgb(209, 247, 245);
}
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>列表</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btNew">
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btNew
{
	border-image: url(:/img/resource/add.png);
}
QPushButton#btNew:hover
{
	border-image: url(:/img/resource/add-hover.png);
}
QPushButton#btNew:pressed
{
	border-image: url(:/img/resource/add-click.png);
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btRemove">
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Highlight">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>51</red>
                  <green>153</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Highlight">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>51</red>
                  <green>153</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Highlight">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>51</red>
                  <green>153</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btRemove
{
	border-image: url(:/img/resource/remove.png);
}
QPushButton#btRemove:hover
{
	border-image: url(:/img/resource/remove-hover.png);
}
QPushButton#btRemove:pressed
{
	border-image: url(:/img/resource/remove-click.png);
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btSearch">
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Highlight">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>51</red>
                  <green>153</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Highlight">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>51</red>
                  <green>153</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Highlight">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>51</red>
                  <green>153</green>
                  <blue>255</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btRemove
{
	border-image: url(:/img/resource/remove.png);
}
QPushButton#btRemove:hover
{
	border-image: url(:/img/resource/remove-hover.png);
}
QPushButton#btRemove:pressed
{
	border-image: url(:/img/resource/remove-click.png);
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableList">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>51</red>
                <green>153</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="HighlightedText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>51</red>
                <green>153</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="HighlightedText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>51</red>
                <green>153</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="HighlightedText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>主题</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>分类</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>创建时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>修改时间</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">QGroupBox#groupBox
{
	border:1px solid gray;
	border-radius:10px;
	background-color: rgb(218, 218, 218);
}
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>主题：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditTopic"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>分类：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxType"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>创建时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCreateTime">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>修改时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelUpdateTime">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>内容：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>25</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btSave">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>保存</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btSave
{
	border-image: url(:/img/resource/save.png);
}
QPushButton#btSave:hover
{
	border-image: url(:/img/resource/save-hover.png);
}
QPushButton#btSave:pressed
{
	border-image: url(:/img/resource/save-click.png);
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btCancel">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>恢复</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btCancel
{
	border-image: url(:/img/resource/refresh.png);
}
QPushButton#btCancel:hover
{
	border-image: url(:/img/resource/refresh-hover.png);
}
QPushButton#btCancel:pressed
{
	border-image: url(:/img/resource/refresh-click.png);
}

</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btExport">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="toolTip">
             <string>导出到文本</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btExport
{
	border-image: url(:/img/resource/export.png);
}
QPushButton#btExport:hover
{
	border-image: url(:/img/resource/export-hover.png);
}
QPushButton#btExport:pressed
{
	border-image: url(:/img/resource/export-click.png);
}

</string>
            </property>
            <property name="text">
             <string>^</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditContent"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
